src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -351,6 +363,77 @@ def get_stats_data_for_trending_events(results):
                 else {},
             }
 
+        def fingerprint_regression(trending_event):
+            return hashlib.sha1(
+                (
+                    f'p95_transaction_duration_regression-{trending_event["transaction"]}-experiment'
+                ).encode()
+            ).hexdigest()
+
+        def send_occurrence_to_plaform(found_trending_events):
+            nonlocal top_trending_transactions
+            qualifying_trends = []
+            for trend in found_trending_events:
+                if (
+                    request.GET.get("statsPeriod", None) == "14d"
+                    and trend.get("change", None) == "regression"
+                    # trends >50%
+                    and (trend.get("trend_percentage", None) - 1) >= 0.5
+                    and trend_function == "p95(transaction.duration)"
+                ):
+                    qualifying_trends.append(trend)
+
+            current_timestamp = datetime.utcnow().replace(tzinfo=timezone.utc)
+            for qualifying_trend in qualifying_trends:
+                displayed_old_baseline = round(float(qualifying_trend["aggregate_range_1"]), 2)
+                displayed_new_baseline = round(float(qualifying_trend["aggregate_range_2"]), 2)
+
+                project_id = next(
+                    transaction["project_id"]
+                    for transaction in top_trending_transactions["data"]
+                    if transaction["transaction"] == qualifying_trend["transaction"]
+                    and transaction["project"] == qualifying_trend["project"]
+                )
+
+                occurrence = IssueOccurrence(
```

Comment:
There should be a metric emitted so we know how often this is happening

Reply:
true, added a metric 👍